storage/mysql/log_storage_test.go

```diff
@@ -977,38 +977,34 @@ func TestGetUnsequencedCounts(t *testing.T) {
 	s := NewLogStorage(DB, nil)
 
 	ctx := context.Background()
-	expectedCount := int64(0)
+	expectedCount := make(map[int64]int64)
 
 	for i := int64(1); i < 10; i++ {
```

Comment:
Suggestion: I would prefer this either as a one-shot test, with a fixed (but easier to read count), or a full table-driven thingy.  The outer loop isn't buying us much.

Eg:


```go
wantCount := map[int64]int64 {
  logs[0]: 0,
  logs[1]: 10,  // All arbitrary
  logs[2]: 5,
  logs[3]: 35,
}
for log, count := range wantCount {
  // queue leaves
}
// Snapshot, query, diff
```